packages/cli/test/project.js

```diff
@@ -0,0 +1,158 @@
+        assert.file([
+          'package.json',
+          '.yo-rc.json',
+          '.prettierrc',
+          '.gitignore',
+          '.npmrc',
+          'tslint.json',
+          'src/index.ts',
+        ]);
+        assert.jsonFileContent('package.json', props);
+        assert.fileContent([
+          ['package.json', '@loopback/build'],
+          ['tslint.json', '@loopback/build'],
+          ['tsconfig.json', '@loopback/build'],
+        ]);
+        assert.noFileContent([
+          ['package.json', '"typescript"'],
+          ['tslint.json', '"rules"'],
+          ['tsconfig.json', '"compilerOptions"'],
+        ]);
+      });
+    });
+
+    describe('with loopbackBuild disabled', () => {
+      before(() => {
+        return helpers.run(appGenerator).withPrompts(
+          Object.assign(
+            {
+              settings: [
+                // Force Enable loopbackBuild to be unchecked
+                'Disable loopbackBuild',
+                'Enable tslint',
+                'Enable prettier',
+                'Enable mocha',
+              ],
+            },
+            props
+          )
+        );
+      });
+
+      it('creates files', () => {
+        assert.jsonFileContent('package.json', props);
+        assert.noFileContent([
+          ['package.json', '@loopback/build'],
+          ['tslint.json', '@loopback/build'],
+          ['tsconfig.json', '@loopback/build'],
+        ]);
+        assert.fileContent([
+          ['package.json', '"typescript"'],
+          ['package.json', '"tslint"'],
+          ['package.json', '"prettier"'],
+          ['tslint.json', '"rules"'],
+          ['tsconfig.json', '"compilerOptions"'],
+        ]);
+      });
+    });
+
+    describe('with prettier disabled', () => {
+      before(() => {
+        return helpers.run(appGenerator).withPrompts(
+          Object.assign(
+            {
+              settings: [
+                'Enable loopbackBuild',
+                'Enable tslint',
+                'Disable prettier', // Force Enable prettier to be unchecked
+                'Enable mocha',
+              ],
+            },
+            props
+          )
+        );
+      });
+
+      it('creates files', () => {
+        assert.noFile(['.prettierrc', '.prettierrcignore']);
+        assert.jsonFileContent('package.json', props);
+      });
+    });
+
+    describe('with tslint disabled', () => {
+      before(() => {
+        return helpers.run(appGenerator).withPrompts(
+          Object.assign(
+            {
+              settings: [
+                'Enable loopbackBuild',
+                'Disable tslint', // Force Enable tslint to be unchecked
+                'Enable prettier',
+                'Enable mocha',
+              ],
+            },
+            props
+          )
+        );
+      });
+
+      it('creates files', () => {
+        assert.noFile(['tslint.json', 'tslint.build.json']);
+        assert.jsonFileContent('package.json', props);
```

Comment:
I wonder how `assert.jsonFileContent` works. Below we are checking for `prettier` [here](https://github.com/strongloop/loopback-next/pull/693/files#diff-626d225f72681fb6a0d8254766ba46e6R151), but not in this particular test?

Reply:
See https://github.com/yeoman/yeoman-assert#api. The assertions here focus on verifying the omission of tslint related artifacts.